src/core/TrainController.cpp

```diff
@@ -431,6 +433,8 @@ namespace godot {
         emit_signal(CONFIG_CHANGED);
     }
 
+    void TrainController::set_config_property(String &key, Variant &p_value) {}
```

Comment:
Why? Is this change related to the code cleanup? The config property is READ-ONLY.

Reply:
Method was declared in .hpp file but not in .cpp

So i just have declared it in .cpp